rcsb/db/mongo/PdbxLoader.py

```diff
@@ -1156,3 +1156,61 @@ def checkValidationDataCount(self, databaseName, collectionName, minValidationCo
         except Exception as e:
             logger.exception("Failing with %s", str(e))
         return ok
+
+    def checkLoadedEntriesWithHoldingsCount(self):
+        """Get the count of documents in the given collection with validation data"""
+        ok = True
+        try:
+            entryCount = 0
+            holdingCount = 0
+            combinedHoldingActualCount = 0
+            combinedHoldingExpectedCount = 0
+
+            exdbCollections = {
+                "pdbx_core": ["pdbx_core_entry"],
+                "repository_holdings": [
+                    "repository_holdings_combined_entry",
+                    "repository_holdings_current_entry",
+                    "repository_holdings_removed_entry",
+                    "repository_holdings_unreleased_entry",
+                    "repository_holdings_update_entry",
```

Comment:
@piehld Where is `repository_holdings_update_entry` used?

Reply:
Good question...I don't see where that collection ever actually gets used (I don't see it passed into DW). But this is helpful to keep in mind in the direction of possibly loading deltas.